### PR TITLE
Small updates

### PR DESCRIPTION
**Pull request by rabernat**
- Add learning goals to indexing
- Mention `align` in indexing
- Use cloud data instead of opendap in computation

### CODE REVIEW

**Comment by rabernat** [comment]
I don't have rights to merge this. I would like to do it asap so the binder can build before the tutorial starts.

Help @jhamman @dcherian @keewis @shoyer!

Also, why is the pre-commit hook failing?

File: scipy-tutorial/02_working_with_labeled_data.ipynb [hunk 1]
```diff
@@ -458,7 +465,19 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a_al, b_al = xr.align(a, b, join=\"inner\")\n",
+    "b_l"
```
Comment (keewis) [inline scipy-tutorial/02_working_with_labeled_data.ipynb:480]:
```suggestion
    "b_al"
```

**Comment by keewis** [comment]
> I don't have rights to merge this. I would like to do it asap so the binder can build before the tutorial starts.

that's because you didn't accept the invitation to become a member of `xarray-contrib`. I triggered a new invitation, hopefully that helps?

**Comment by keewis** [comment]
unless you have more changes I'll merge on green

**Comment by rabernat** [comment]
Sorry about that.

**Pull request merged**